box-tool/src/main/java/smashdudes/particletool/presentation/EmitterEditorWidget.java

```diff
@@ -27,6 +27,10 @@ protected void draw(ShapeRenderer sh, SpriteBatch sb)
         if(ImGui.button("Play"))
         {
             context.play();
+            if(context.isFinished())
+            {
+                context.reset();
+            }
         }
```

Comment:
Maybe signal that the emission is done?

```suggestion
        String text = context.isFinished() ? "Reset" : "Play"
        if(ImGui.button(text))
        {
            if(context.isFinished())
            {
                context.reset();
            }
            else 
            {
                context.play();
            }
```